pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
One could also split the `str` generation to it's own mixin in `string.py` and let both Series and index take that...